pkg/client/unversioned/util.go

```diff
@@ -77,3 +79,13 @@ func RetryOnConflict(backoff wait.Backoff, fn func() error) error {
 	}
 	return err
 }
+
+func isEnabledVersion(group string, gv *unversioned.GroupVersion) bool {
```

Comment:
Why not use registered.IsEnabledVersion() directly? https://github.com/kubernetes/kubernetes/blob/master/pkg/apimachinery/registered/registered.go#L137